projects/pt1/python/torch_mlir_e2e_test/test_suite/elementwise.py

```diff
@@ -3463,6 +3490,58 @@ def ElementwiseAtenIsinfOpModule_basic(module, tu: TestUtils):
 # ==============================================================================
 
 
+class ElementwiseAtenIsneginfOpModule(torch.nn.Module):
+    def __init__(self):
+        super().__init__()
+
+    @export
+    @annotate_args([
+        None,
+        ([2, 5], torch.float32, True),
+    ])
+    def forward(self, x):
+        return torch.ops.aten.isneginf(x)
+
+@register_test_case(module_factory=lambda: ElementwiseAtenIsneginfOpModule())
+def ElementwiseAtenIsneginfOpModule_basic(module, tu:TestUtils):
+    test_input = torch.tensor(
+        [
+            [1, float('-inf'), 2, float('-inf'), float('nan')],
+            [1, float('-inf'), float('-inf'), float('nan'), 3],
```

Comment:
change some `-inf` to `inf` to make this test more robust? Same as test of `aten.isposinf`.